metadata-ingestion/src/datahub/ingestion/source/looker_common.py

```diff
@@ -462,6 +462,11 @@ def _get_fields_and_primary_keys(
                 primary_keys.append(schema_field.fieldPath)
         return fields, primary_keys
 
+    @staticmethod
+    def _display_name(name: str) -> str:
```

Comment:
Do we usually use underscore at the beginning of "public" method names?

Reply:
did it this way to align with the other methods ... will come back and clean up this class later.